development/build/manifest.js

```diff
@@ -3,7 +3,10 @@ const path = require('path');
 const childProcess = require('child_process');
 const { mergeWith, cloneDeep } = require('lodash');
 
-const baseManifest = process.env.ENABLE_MV3
+const IS_MV3_ENABLED =
+  process.env.ENABLE_MV3 === 'true' || process.env.ENABLE_MV3 === undefined;
+
```

Comment:
Since we have to check this condition so many times, can we move this code to a common file, and export the value there?

Maybe in `development/build/manifest.js`?